html-links/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Links</title>
+  </head>
+  <body>
+    <h1>Popular Search Engines</h1>
+    <ul>
+      <a href="https://www.google.com/" target="_blank">
+        <li>Google</li>
+      </a>
+      <a href="https://www.yahoo.com/?guccounter=1">
+        <li>Yahoo</li>
+      </a>
+      <a href="https://www.bing.com/">
+        <li>Bing</li>
+      </a>
+      <a href="https://www.ask.com/">
+        <li>Ask</li>
+      </a>
+      <a href="https://duckduckgo.com/">
+        <li>Duck Duck Go</li>
+      </a>
+    </ul>
```

Comment:
The only valid children of a `ul` are `li` elements. You cannot have any other elements be a direct child to a `ul`